static/home/css/style.css

```diff
@@ -240,12 +248,21 @@ a:hover{
 }
 #about-us .container div{
 	max-height: 500px;
+	align-items: center;
+	justify-content: center;
+	word-spacing: .1em;
+}
+#about-us p {
+	font-size: 1rem;
+	width: 90%;
+	margin: 0 auto;
 }
 .vision{
 	display: grid;
 	grid-template-columns: 1fr 1fr;
 	grid-column-gap: 1em;
 	padding: 20px 0;
+	margin-bottom: 10px;
```

Comment:
Instead of here, I think `margin-bottom` property would be better in `#about-us .container div`, as it will automatically apply to all the divs in `#about-us` section.

Reply:
removing it.